aiida/orm/utils/managers.py

```diff
@@ -142,6 +148,25 @@ def __getattr__(self, name):
                 f"Node<{self._node.pk}> does not have an {prefix} with link label '{name}'"
             ) from exception
 
+    def __contains__(self, key):
+        """Override the operator of the base class to emit deprecation warning if double underscore is used in key."""
+        if self._namespace_separator in key:
+            warnings.warn(
+                'The use of double underscores in keys is deprecated. Please expand the namespaces manually.',
```

Comment:
"Please expand the name manually" is correct but I'm not sure it will be clear enough for people who don't have the context of this change? I would maybe add an example like in the warning above.

```suggestion
                'The use of double underscores in keys is deprecated. Please expand the namespaces manually: '
                'instead of `nested__name in node.inputs`, use `nested in node.inputs` first and `name in node.inputs.nested` afterwards',
```

Reply:
Thanks I have included a slightly different variant of your suggestion